docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -547,38 +409,19 @@ in the metastore. As a result, any Databricks engine can write to the table::
 The Delta Lake connector also supports creating tables using the :doc:`CREATE
 TABLE AS </sql/create-table-as>` syntax.
 
-The following properties are available for use:
-
-.. list-table:: Delta Lake table properties
-  :widths: 40, 60
-  :header-rows: 1
-
-  * - Property name
-    - Description
-  * - ``location``
-    - File system location URI for the table.
-  * - ``partitioned_by``
-    - Set partition columns.
-  * - ``checkpoint_interval``
-    - Set the checkpoint interval in seconds.
-  * - ``change_data_feed_enabled``
-    - Enables storing change data feed entries.
+Procedures
+^^^^^^^^^^
 
-The following example uses all available table properties::
+Use the :doc:`/sql/call` statement to perform data manipulation or
+administrative tasks. Procedures must include a qualified catalog name, if your
```

Comment:
Remove the "Procedures.." sentence and replace with something like

Procedures are available in the system schema of each catalog. The following code display how to call the example_procedure in the example catalog.